build/Linux/opt/devatserv/bin/start-devatserv.sh

```diff
@@ -9,45 +9,49 @@ DAS_GUI_DIR="/opt/share/applications/DevAtServGUI_1.0.0_amd64.deb"
 
 pre_check_installation() {
   echo -e "${MSG_INFO} Starting pre-check-installation"
+  local err=0
 
   if ! command -v docker &> /dev/null; then
+    err=1
     echo -e "${MSG_ERR} Failed to find 'docker'"
```

Comment:
Hi Thong,
as Docker is mandatory you just log "installing Docker" and execute installation. No choice, no question.
In case of Error you log an error message. In this case the whole installation fails as we are dependent on Docker.
Thank you,
 Thomas

Reply:
Hi Thomas,

I have updated as your mentions. 
Please help me review it.

Thank you and best regards,
Thong Hua